src/nspawn/nspawn-mount.c

```diff
@@ -989,6 +989,20 @@ int mount_custom(
         return 0;
 }
 
+bool has_root_mount(CustomMount *mounts, size_t n) {
```

Comment:
maybe call this function has_custom_root_mount() maybe? after all we'll have a root mount either way, key is whether it is custom